src/webview/editConnection.js

```diff
@@ -1,11 +1,8 @@
-// TODO Update implementation to use javadocs
-// @ts-nocheck
-
 // This script will be run within the webview itself
 // It cannot access the main VS Code APIs directly.
+
 (function () {
   const vscode = acquireVsCodeApi();
-
   const oldState = /** @type {{ oldConnectionName: string} | undefined} */ vscode.getState();
```

Comment:
Seems like the VSCode API that's exposed within the webview isn't consistent; I don't mind occasional type-assertions like this as long as they're used sparingly and keep development from getting bogged down on placating the type-checker.